bin/echo.c

```diff
@@ -27,6 +27,48 @@
 #include <errno.h>
 
 #include <s2n.h>
+#include "crypto/s2n_rsa.h"
+#include "stuffer/s2n_stuffer.h"
+#include "utils/s2n_safety.h"
```

Comment:
I had always thought of stuffers as a separate library that could be used externally. Not as sure about the safety checks though.